webpack/assets/javascripts/react_app/components/Layout/__tests__/__snapshots__/LayoutSelectors.test.js.snap

```diff
@@ -98,17 +98,19 @@ Array [
         "className": "",
         "href": "/",
         "isDivider": false,
-        "onClick": null,
+        "onClick": undefined,
```

Comment:
@ohadlevy the changes here are only from `null` to `undefined` and adding url for each child, seems ok.